### ISSUE TITLE
training miou 47~48

### ISSUE THREAD

**Issue by mathfinder** [anilbatra2185/road_connectivity#26]
Is it corre that the first 30 epochs keep miou 47~48？
![image](https://user-images.githubusercontent.com/23050243/77217783-11ed0580-6b60-11ea-9edc-d43f9a8336b6.png)

**Comment by mathfinder** [comment]
![image](https://user-images.githubusercontent.com/23050243/77218314-f5070100-6b64-11ea-8cce-ef8682417899.png)
I just migrated it to torch 1.4. It looks not something wrong.

**Comment by anilbatra2185** [comment]
Hi @mathfinder, 

thanks for working on it and migrating to version 1.4.  Apologies for delayed response.

In my training process miou stays around ~ 0.47-0.48 for first few epochs (~ 5-8 epochs) and after that it start improving. Additionally, if you are willing to share your github link or code then I can help. 

Thanks
Anil

**Comment by zzzbosss1997** [comment]
You should rewrite the loss function and the loss backward on 1.4 or later version
